gcc version 6.3.0 unloaded.
gcc version 9.1.0 loaded.
openmpi version 4.0.4 loaded.
Python 3.8.5 with TensorFlow 2.3.0 loaded.
Traceback (most recent call last):
  File "DeepChromeHiC.py", line 157, in <module>
    mkdir('result/' + args.name + '/onehot_cnn_one_branch')
  File "DeepChromeHiC.py", line 146, in mkdir
    os.makedirs(path) # makedirs 创建文件时如果路径不存在会创建这个路径
  File "/N/soft/rhel7/deeplearning/Python-3.8.5/lib/python3.8/os.py", line 223, in makedirs
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: 'result/AD2.pp/onehot_cnn_one_branch'
WARNING:tensorflow:From /N/slate/bizi/DeepChromeHiC/train.py:335: Model.fit_generator (from tensorflow.python.keras.engine.training) is deprecated and will be removed in a future version.
Instructions for updating:
Please use Model.fit, which supports generators.
WARNING:tensorflow:From /N/slate/bizi/DeepChromeHiC/train.py:184: Model.fit_generator (from tensorflow.python.keras.engine.training) is deprecated and will be removed in a future version.
Instructions for updating:
Please use Model.fit, which supports generators.
WARNING:tensorflow:From /N/slate/bizi/DeepChromeHiC/train.py:184: Model.fit_generator (from tensorflow.python.keras.engine.training) is deprecated and will be removed in a future version.
Instructions for updating:
Please use Model.fit, which supports generators.
WARNING:tensorflow:Gradients do not exist for variables ['basic_block/sequential_1/conv2d_3/kernel:0', 'basic_block/sequential_1/conv2d_3/bias:0', 'basic_block/sequential_1/batch_normalization_3/gamma:0', 'basic_block/sequential_1/batch_normalization_3/beta:0', 'basic_block_2/sequential_3/conv2d_8/kernel:0', 'basic_block_2/sequential_3/conv2d_8/bias:0', 'basic_block_2/sequential_3/batch_normalization_8/gamma:0', 'basic_block_2/sequential_3/batch_normalization_8/beta:0', 'basic_block_4/sequential_5/conv2d_13/kernel:0', 'basic_block_4/sequential_5/conv2d_13/bias:0', 'basic_block_4/sequential_5/batch_normalization_13/gamma:0', 'basic_block_4/sequential_5/batch_normalization_13/beta:0', 'basic_block_6/sequential_7/conv2d_18/kernel:0', 'basic_block_6/sequential_7/conv2d_18/bias:0', 'basic_block_6/sequential_7/batch_normalization_18/gamma:0', 'basic_block_6/sequential_7/batch_normalization_18/beta:0'] when minimizing the loss.
WARNING:tensorflow:Gradients do not exist for variables ['basic_block/sequential_1/conv2d_3/kernel:0', 'basic_block/sequential_1/conv2d_3/bias:0', 'basic_block/sequential_1/batch_normalization_3/gamma:0', 'basic_block/sequential_1/batch_normalization_3/beta:0', 'basic_block_2/sequential_3/conv2d_8/kernel:0', 'basic_block_2/sequential_3/conv2d_8/bias:0', 'basic_block_2/sequential_3/batch_normalization_8/gamma:0', 'basic_block_2/sequential_3/batch_normalization_8/beta:0', 'basic_block_4/sequential_5/conv2d_13/kernel:0', 'basic_block_4/sequential_5/conv2d_13/bias:0', 'basic_block_4/sequential_5/batch_normalization_13/gamma:0', 'basic_block_4/sequential_5/batch_normalization_13/beta:0', 'basic_block_6/sequential_7/conv2d_18/kernel:0', 'basic_block_6/sequential_7/conv2d_18/bias:0', 'basic_block_6/sequential_7/batch_normalization_18/gamma:0', 'basic_block_6/sequential_7/batch_normalization_18/beta:0'] when minimizing the loss.
WARNING:tensorflow:Gradients do not exist for variables ['basic_block/sequential_1/conv2d_3/kernel:0', 'basic_block/sequential_1/conv2d_3/bias:0', 'basic_block/sequential_1/batch_normalization_3/gamma:0', 'basic_block/sequential_1/batch_normalization_3/beta:0', 'basic_block_2/sequential_3/conv2d_8/kernel:0', 'basic_block_2/sequential_3/conv2d_8/bias:0', 'basic_block_2/sequential_3/batch_normalization_8/gamma:0', 'basic_block_2/sequential_3/batch_normalization_8/beta:0', 'basic_block_4/sequential_5/conv2d_13/kernel:0', 'basic_block_4/sequential_5/conv2d_13/bias:0', 'basic_block_4/sequential_5/batch_normalization_13/gamma:0', 'basic_block_4/sequential_5/batch_normalization_13/beta:0', 'basic_block_6/sequential_7/conv2d_18/kernel:0', 'basic_block_6/sequential_7/conv2d_18/bias:0', 'basic_block_6/sequential_7/batch_normalization_18/gamma:0', 'basic_block_6/sequential_7/batch_normalization_18/beta:0'] when minimizing the loss.
WARNING:tensorflow:Gradients do not exist for variables ['basic_block/sequential_1/conv2d_3/kernel:0', 'basic_block/sequential_1/conv2d_3/bias:0', 'basic_block/sequential_1/batch_normalization_3/gamma:0', 'basic_block/sequential_1/batch_normalization_3/beta:0', 'basic_block_2/sequential_3/conv2d_8/kernel:0', 'basic_block_2/sequential_3/conv2d_8/bias:0', 'basic_block_2/sequential_3/batch_normalization_8/gamma:0', 'basic_block_2/sequential_3/batch_normalization_8/beta:0', 'basic_block_4/sequential_5/conv2d_13/kernel:0', 'basic_block_4/sequential_5/conv2d_13/bias:0', 'basic_block_4/sequential_5/batch_normalization_13/gamma:0', 'basic_block_4/sequential_5/batch_normalization_13/beta:0', 'basic_block_6/sequential_7/conv2d_18/kernel:0', 'basic_block_6/sequential_7/conv2d_18/bias:0', 'basic_block_6/sequential_7/batch_normalization_18/gamma:0', 'basic_block_6/sequential_7/batch_normalization_18/beta:0'] when minimizing the loss.
